Latest ./run-in-docker fails build tests with "Unable to open socket file..."

Following the instructions (exactly) in the readme results in the following error(s):

    testVerbose(io.swagger.codegen.config.CodegenConfiguratorTest)  Time elapsed: 5.216 sec  <<< FAILURE!
    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
        at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:205)
        at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:108)
        at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
    Caused by: java.lang.reflect.InvocationTargetException: null
        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
        at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
        at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:205)
        at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:108)
        at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
    Caused by: com.sun.tools.attach.AttachNotSupportedException: Unable to open socket file: target process not responding or HotSpot VM not loaded
        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
        at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
        at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:205)
        at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:108)
        at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)


    Results :

    Failed tests:
      CodegenConfiguratorTest.testAdditionalProperties » Runtime java.lang.reflect.I...
      CodegenConfiguratorTest.testApiPackage » Runtime java.lang.reflect.InvocationT...
      CodegenConfiguratorTest.testArtifactId » Runtime java.lang.reflect.InvocationT...
      CodegenConfiguratorTest.testArtifactVersion » Runtime java.lang.reflect.Invoca...
      CodegenConfiguratorTest.testDynamicProperties » Runtime java.lang.reflect.Invo...
      CodegenConfiguratorTest.testFromFile » Runtime java.lang.reflect.InvocationTar...
      CodegenConfiguratorTest.testGroupId » Runtime java.lang.reflect.InvocationTarg...
      CodegenConfiguratorTest.testImportMappings » Runtime java.lang.reflect.Invocat...
      CodegenConfiguratorTest.testInstantiationTypes » Runtime java.lang.reflect.Inv...
      CodegenConfiguratorTest.testInvokerPackage » Runtime java.lang.reflect.Invocat...
      CodegenConfiguratorTest.testLanguageSpecificPrimitives » Runtime java.lang.ref...
      CodegenConfiguratorTest.testLibrary » Runtime java.lang.reflect.InvocationTarg...
      CodegenConfiguratorTest.testModelPackage » Runtime java.lang.reflect.Invocatio...
      CodegenConfiguratorTest.testSkipOverwrite » Runtime java.lang.reflect.Invocati...
      CodegenConfiguratorTest.testSystemProperties » Runtime java.lang.reflect.Invoc...
      CodegenConfiguratorTest.testTemplateDir » Runtime java.lang.reflect.Invocation...
      CodegenConfiguratorTest.testTypeMappings » Runtime java.lang.reflect.Invocatio...
      CodegenConfiguratorTest.testVerbose » Runtime java.lang.reflect.InvocationTarg...

    Tests run: 120, Failures: 18, Errors: 0, Skipped: 0

    [INFO] ------------------------------------------------------------------------
    [INFO] Reactor Summary:
    [INFO]
    [INFO] swagger-codegen-project ............................ SUCCESS [ 11.895 s]
    [INFO] swagger-codegen (core library) ..................... FAILURE [04:30 min]
    [INFO] swagger-codegen (executable) ....................... SKIPPED
    [INFO] swagger-codegen (maven-plugin) ..................... SKIPPED
    [INFO] swagger-generator .................................. SKIPPED
    [INFO] ------------------------------------------------------------------------
    [INFO] BUILD FAILURE
    [INFO] ------------------------------------------------------------------------
    [INFO] Total time: 04:48 min
    [INFO] Finished at: 2015-10-02T17:07:05+00:00
    [INFO] Final Memory: 38M/213M
    [INFO] ------------------------------------------------------------------------
    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.18.1:test (test-testng) on project swagger-codegen: There are test failures.
    [ERROR]
    [ERROR] Please refer to /gen/modules/swagger-codegen/target/surefire-reports for the individual test results.
    [ERROR] -> [Help 1]
    [ERROR]
    [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
    [ERROR] Re-run Maven using the -X switch to enable full debug logging.
    [ERROR]
    [ERROR] For more information about the errors and possible solutions, please read the following articles:
    [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
    [ERROR]
    [ERROR] After correcting the problems, you can resume the build with the command
    [ERROR]   mvn <goals> -rf :swagger-codegen

~/W/j/swagger-codegen git:master ❯❯❯ docker -v
Docker version 1.8.1, build d12ea79
Your docs mention installing Java 7 on OSX, but this is irrelevant when running inside the docker container, no?

--- a/pom.xml
+++ b/pom.xml
@@ -74,6 +74,7 @@
                 <version>${surefire-version}</version>
                 <configuration>
                     <testNGArtifactName>none:none</testNGArtifactName>
+                    <argLine>-XX:+StartAttachListener</argLine>
                 </configuration>
                 <executions>
                     <execution>

